includes/transformer/class-factory.php

```diff
@@ -7,6 +7,7 @@
 
 namespace Activitypub\Transformer;
 
+use Activitypub\Activity\Base_Object;
```

Comment:
Is that possible? I thought you have to always use the full path in the check!?!

Reply:
I think only if we were to pass it as a string. But I'm happy to move it inline to be safe.

Reply:
no no, I am just curious!!!